ModelUploader/README.md

```diff
@@ -1,3 +1,7 @@
+# NOTICE! THIS TOOL WILL BE DEPRECATED ON AUGUST 1st 2021
```

Comment:
Do you mean deprecated (still available but marked as obsolete) or removed on August 1st?